nengo_spa/vector_generation.py

```diff
@@ -90,6 +90,57 @@ def next(self):
         return self.__next__()
 
 
+class EquallySpacedPositiveUnitaryHrrVectors:
+    """Generator for equally spaced positive unitary HRR vectors.
+
+    The vectors produced by this generator lie all on the hyper-circle of
```

Comment:
Very minor thing, but I think it would be more clear to say "on a hyper-circle" rather than "on the hyper-circle" since this generates vectors across just one of many possible hyper-circles of positive, unitary vectors for the given dimensionality.